core/data/src/main/java/org/mifos/mobilewallet/core/data/fineract/entity/mapper/AccountMapper.kt

```diff
@@ -0,0 +1,31 @@
+package org.mifos.mobilewallet.core.data.fineract.entity.mapper
+
+import com.mifos.mobilewallet.model.entity.accounts.savings.SavingAccount
+import com.mifos.mobilewallet.model.entity.client.ClientAccounts
+import com.mifos.mobilewallet.model.domain.Account
+import javax.inject.Inject
+
+class AccountMapper @Inject constructor(private val currencyMapper: CurrencyMapper) {
+
+    fun transform(clientAccounts: ClientAccounts?): List<Account> {
+        val accountList = mutableListOf<Account>()
+
+        if (clientAccounts != null && clientAccounts.savingsAccounts != null
```

Comment:
We can combine these two `clientAccounts.savingsAccounts != null
            && clientAccounts.savingsAccounts.isNotEmpty()`
            
  `!clientAccounts.savingsAccounts.isNullOrEmpty()`

Reply:
done